docs/releases/1.11.7.txt

```diff
@@ -19,3 +19,8 @@ Bugfixes
 
 * Fixed unquoted table names in ``Subquery`` SQL when using ``OuterRef``
   (:ticket:`28689`).
+
+* Lookups for :class:`~django.contrib.postgres.fields.CICharField`,
```

Comment:
You can remove the links for `CICharField`, etc. and just use the class names. That's to avoid having to update all the minor release notes in case things mentioned in them are later deprecated and removed.